types/TestRunner.js

```diff
@@ -18,14 +18,16 @@ import type {
   TestResult,
 } from 'types/TestResult';
 import type Runtime from 'jest-runtime';
-import type TestWatcher from 'jest-cli/src/test_watcher';
+import type {TestWatcher as _TestWatcher} from 'jest-cli';
```

Comment:
Can't we just keep it with the standard name? I don't see any place where it could collide.